Add new Kitworks migration prerequisites (#1397)
owncloud/docs-server: modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

=== Kiteworks

* The following major prerequisites apply:

** The Kiteworks instance *must* be running version 8.6 or higher.
** Kiteworks requires a `Data migrator` license.

* You need to login into the Kiteworks appliance as role *System Admin*.
// The kiteworks satellite service must be activated and available to the system admin user account.

* The Kiteworks system must provide sufficient disk space for the data to be migrated. The ownCloud xref:migration-verification[occ migrate:to-kiteworks:verify] step will report the estimated disk space needed.

* Ensure quota is set to unlimited in the Kiteworks user profiles.

> * Set the web application firewall (WAF) to report only and not to block file upload during migration. Kiteworks support will help to configure the WAF.

Reviewer: ```suggestion
* Set the web application firewall (WAF) to `report only` so that file uploads are not blocked during migration. Kiteworks support will help to configure the WAF.
```

=== Kiteworks

* The following major prerequisites apply:

** The Kiteworks instance *must* be running version 8.6 or higher.

> ** Kiteworks requires a license that includes `data migrator enabled`.

Reviewer: - ' enabled'

Author: fixed

=== Kiteworks

* The following major prerequisites apply:

** The Kiteworks instance *must* be running version 8.6 or higher.
** Kiteworks requires a `Data migrator` license. +

> The respective `Include feature` checkbox must be set to enabled.

Reviewer: lets just drop this line - end users have no understanding of the license generation.
and people generating licenses know what to do aka not our job to document this.

* You need to login into the Kiteworks appliance as role *System Admin*.
// The kiteworks satellite service must be activated and available to the system admin user account.

* The Kiteworks system must provide sufficient disk space for the data to be migrated. The ownCloud xref:migration-verification[occ migrate:to-kiteworks:verify] step will report the estimated disk space needed.

* Ensure sufficient quota settings in the Kiteworks user profiles:

> ** Users have a default quota of 2GB. +

Reviewer: drop the 2gb default - this was the case on my machine - but who knows if this changes under what ever circumstances. furthermore this is of 0 interest - important is that we need the quota unlimited

Before starting the migration, go to the user profile and set the quota to unlimited.

> * Set the web application firewall (WAF) to `report only` and not to `block file upload during migration`. Kiteworks support will help to configure the WAF.

Reviewer: why the ``  around "block file upload ..." - this read just wrong
